src/cf-typography/usage.md

```diff
@@ -50,42 +50,39 @@ Default color variables are from 18F's
 
 ```
 // .pull-quote
-@pull-quote_body:               #494440; // $color-gray-warm-dark
+@pull-quote:               #494440; // $color-gray-warm-dark
 @pull-quote_citation:           #205493; // $color-cool-blue
 
 // .micro-copy
-@micro-copy-text:               #112e51; // $color-primary-darkest
+@micro-copy:               #112e51; // $color-primary-darkest
 
 // .date
-@date-text:                     #112e51; // $color-primary-darkest
+@date:                      #112e51; // $color-primary-darkest
 
 // .category-slug
-@category-slug-text:            #112e51; // $color-primary-darkest
-@category-slug-hover:           #205493; // $color-primary-darker
+@category-slug:             #112e51; // $color-primary-darkest
+@category-slug__hover:      #205493; // $color-primary-darker
 
 // .header-slug
-@header-slug-thin-border:       #323a45; // $color-gray-dark
-@header-slug-thick-border:      #02bfe7; // $color-primary-alt
+@header-slug-border__thin:  #323a45; // $color-gray-dark
+@header-slug-border__thick: #02bfe7; // $color-primary-alt
 
 // .padded-header
-@padded-header-text:            #046b99; // $color-primary-alt-darkest
-@padded-header-bg:              #dce4ef; // color-gray-cool-light
-@padded-header-border:          #046b99; // $color-primary-alt-darkest
+@padded-header:             #046b99; // $color-primary-alt-darkest
+@padded-header_bg:          #dce4ef; // color-gray-cool-light
+@padded-header_border:      #046b99; // $color-primary-alt-darkest
 
 // .fancy-slug
-@fancy-slug-text:               #046b99; // $color-primary-alt-darkest
-@fancy-slug-bg:                 #dce4ef; // color-gray-cool-light
-@fancy-slug-border:             #046b99; // $color-primary-alt-darkest
+@fancy-slug:                #046b99; // $color-primary-alt-darkest
+@fancy-slug_bg:             #dce4ef; // color-gray-cool-light
+@fancy-slug_border:         #046b99; // $color-primary-alt-darkest
 
 // .meta-header
-@meta-header-border:            #323a45; // $color-gray-dark
+@meta-header_border:        #323a45; // $color-gray-dark
 
 // .jump-link
-@jump-link__bg:                 #f1f1f1; // $color-gray-lightest
-@jump-link__bg-border:          #323a45; // $color-gray-dark
-
-// .list__branded
-@list__branded-bullet:          #02bfe7; // $color-primary-alt
+@jump-link_bg:              #f1f1f1; // $color-gray-lightest
+@jump-link_border:          #323a45; // $color-gray-dark
```

Comment:
I'm not the biggest fan of doing a BEM-ish nomenclature for variable names, but if we do, shouldn't we keep it consistent with the class names things apply to? e.g., `pull-quote_body` should remain, since it applies to the element with that class (although the class is now prefixed with `m-`, and I certainly wouldn't want to add the prefixes to all variables...)